### PR TITLE
Explain how to import the effects

### PR DESCRIPTION
**Pull request by blasten**

### CODE REVIEW

File: app-header/app-header.html [hunk 1]
```diff
@@ -83,7 +83,7 @@
 ```
 
 ## Backgrounds
-app-header has two background layers that can be used for styling when the header is condensed 
+app-header has two background layers that can be used for styling when the header is condensed
 or when the scrollable element is scrolled to the top.
 
 ## Scroll effects
```
Comment (keanulee) [inline app-header/app-header.html]: `Scroll effects are _optional_ visual effects...`

File: app-header/app-header.html [hunk 2]
```diff
@@ -98,6 +98,12 @@
 </app-header>
 ```
 
+#### Importing the effects
```
Comment (keanulee) [inline app-header/app-header.html]:
```
#### Using the effects

To use these optional effects, you must explicitly import them in addition to `app-header`:
```

**Comment by blasten** [comment]
done

**Comment by frankiefu** [comment]
LGTM

**Pull request merged**